### PR TITLE
Sms 25 keyboard shortcut for tools

### PR DESCRIPTION
**Pull request by JohnNStout**
This is a combination of SMS 25 and 24. Shortcuts for tools and colors. I also added a shortcut for increasing and decreasing the cursor size.

### CODE REVIEW

**Review by JacobSletten — APPROVED** [review]
Looks good to me.

File: mytestvueapp.client/src/views/PainterView.vue [hunk 1]
```diff
@@ -241,6 +241,30 @@ function ResetArt() {
   router.push("/new");
 }
 
+document.addEventListener("keydown", function (event) { 
+if (event.key === "p") {event.preventDefault();cursor.value.selectedTool.label = "Pan";canvas?.value.updateCursor()} 
+else if (event.key === "b") {event.preventDefault();cursor.value.selectedTool.label = "Brush";canvas?.value.updateCursor()} 
+else if (event.key === "e") {event.preventDefault();cursor.value.selectedTool.label = "Eraser";canvas?.value.updateCursor()} 
+else if (event.key === "d") {event.preventDefault();cursor.value.selectedTool.label = "Pipette";canvas?.value.updateCursor()} 
+else if (event.key === "f") {event.preventDefault();cursor.value.selectedTool.label = "Paint-Bucket";canvas?.value.updateCursor()} 
+else if (event.key === "1") {event.preventDefault();cursor.value.color = "#000000";}
```
Comment (GranquistR) [inline mytestvueapp.client/src/views/PainterView.vue:250]: Use your getDefaultColors function here instead of hard coding

**Review by GranquistR — COMMENTED** [review]
Everything else looks good

File: mytestvueapp.client/src/views/PainterView.vue (hunk 1 above)
Comment (JohnNStout) [reply mytestvueapp.client/src/views/PainterView.vue:250]: fixed.

**Review by GranquistR — APPROVED** [review]

**Pull request merged**